superbench/benchmarks/micro_benchmarks/kernel_launch_overhead/CMakeLists.txt

```diff
@@ -2,10 +2,57 @@
 # Licensed under the MIT License.
 
 cmake_minimum_required(VERSION 3.18)
-project(kernel_launch_overhead LANGUAGES CUDA CXX)
 
-include(../cuda_common.cmake)
+project(kernel_launch_overhead LANGUAGES CXX)
```

Comment:
In my mind, we should a separate PR which superbench should have cuda_common.cmake and rocm_common.cmake. For every benchmark, it should call both cmake directly.

Reply:
Thanks for your suggestion, I have submitted another pr.